portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -801,13 +796,22 @@ else if ("always".equals(showHelpGlobal))
 				
 		boolean showPresence;
 		String globalShowPresence = ServerConfigurationService.getString("display.users.present","true");
-				
+		boolean hidePresenceTool = false;
+
+		try {
+			hidePresenceTool = site.getProperties().getBooleanProperty("presenceTool");
```

Comment:
Well I realized the global property has four different possible values, always, never, true, or false. I adjusted it so that if the administrators set this global property to be always, never, or false, that the tool's visibility depends on that setting. Otherwise, if it's true, it will default to visible, and the instructors will be able to go into the setting to disable the tool from their site, and it differentiates from the global property in the sense that user-personal settings can't be global.